src/utils/preview.ts

```diff
@@ -0,0 +1,16 @@
+import * as vscode from "vscode";
+
+export async function preview(uri: vscode.Uri, column: vscode.ViewColumn, tabTitle: string): Promise<void> {
+    const html = await getHTML(uri);
+    const w = vscode.window.createWebviewPanel('vsk8s-preview', tabTitle, column, {
+        retainContextWhenHidden: false,
+        enableScripts: false,
+    });
+    w.webview.html = html;
+    w.reveal();
+}
+
+async function getHTML(uri: vscode.Uri): Promise<string> {
```

Comment:
it would be nice to move all of the HTML snippets (eventually) to files (or file fragments) in the extension itself...

Reply:
Yes, though a lot of them use JavaScript string interpolation so I'm not sure how feasible that will be.  Definitely out of scope for this PR which is quite scary enough already!